tests/functional/algos/test_algos.py

```diff
@@ -71,9 +71,9 @@ def test_simple(monkeypatch, config_file):
 
 @pytest.mark.usefixtures("clean_db")
 @pytest.mark.usefixtures("null_db_instances")
-@pytest.mark.parametrize('config_file', config_files)
-def test_random_stop(monkeypatch, config_file):
+def test_random_stop(monkeypatch):
     """Test a simple usage scenario."""
+    config_file = 'random_config.yaml'
```

Comment:
Why did you remove the parametrization? This means only random search will be tested.

Reply:
Tpe should stop this case once we get discrete space supported later. now this case is not applied to tpe algorithm.

Reply:
ok. Can you please add a TODO not just to make sure we don't miss it in next PR.